src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/ProjectEngineFactory.cs

```diff
@@ -31,6 +37,11 @@ internal abstract class ProjectEngineFactory : IProjectEngineFactory
 
         return RazorProjectEngine.Create(configuration, fileSystem, b =>
```

Comment:
I know it's a pre-existing name but can we change "b" to something more informative? I assume it would be `builder` based on context.